README.md

```diff
@@ -1 +1,61 @@
-# Module1-Hello-Microverse
+# Project Name
+Hello microverseinc
+
+## Built With
+- Html 
+- Css
+
+## Live Demo (if available)
+
+[Live Demo Link](https://ebukanweje.github.io/Module1-Hello-Microverse/)
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```

Comment:
- [ ] [OPTIONAL] You can remove this for now, since you don't need it for this project 😄